mobile_payment/apps.py

```diff
@@ -0,0 +1,49 @@
+from django.apps import AppConfig
+
+MODULE_NAME = "mobile_payment"
+
+DEFAULT_CFG = {
+      
+      "default_validations_disabled":False,
+      "gql_query_payment_transaction_perms":["125000"],
```

Comment:
those codes are already taken. Please check the [reserved ones](https://openimis.atlassian.net/wiki/spaces/OP/pages/1775173658/openIMIS+Authorities) and take another range.